### ISSUE TITLE
No Issue, Questions

### ISSUE THREAD

**Issue by imjollyroger** [RickSayland/DispoInventoryChecker#1]
Hey man, you've also done yourself some poking on the TL API I see. I'm curious to know if you've tried mapping any additional actions for other things. I'd love to get a list of all the inventory at a location but i haven't been able to hit any combo of parameters that makes that happen. Right now, I have some JS that can scrape the page for inventory URLs, scrape those for the SKUs and then update do a similar API request for the inventory of that item across multiple locations. Would love to find an easier way obviously.

**Comment by RickSayland** [comment]
Yeah I ran out of momentum because I would need to come up with all the
SKUs and didn't feel like figuring out how to scrape them.


As far as I know, I saw it work the other way around. You can get all the
locations for a particular sku but not all the sku's from a location.
Bummer.
